diagnostic.md

```diff
@@ -54,7 +66,10 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big
+#contentPane
+.important.red
+div p
 <!-- your answer ends here -->
```

Comment:
div p gets all p tags that are down the family tree of div, but to be more specific and get direct children of a selector, you'd use div > p